pip3 install error due to plyvel version

* Trinity version: trinity-0.1.0a29
* OS: osx
* Python version (output of `python --version`): 3.6.1
* Environment (output of `pip freeze`):
```
aiohttp==3.6.0
appnope==0.1.0
argcomplete==1.10.0
asn1crypto==1.2.0
async-exit-stack==1.0.1
async-generator==1.10
async-lru==0.1.0
async-timeout==3.0.1
asyncio-cancel-token==0.2.0
attrdict==2.0.1
attrs==19.3.0
backcall==0.1.0
base58==1.0.3
bloom-filter==1.3
cached-property==1.5.1
cachetools==3.1.1
certifi==2019.9.11
cffi==1.13.1
chardet==3.0.4
coincurve==10.0.0
cryptography==2.6.1
cytoolz==0.10.0
decorator==4.4.0
eth-abi==1.3.0
eth-account==0.2.3
eth-bloom==1.0.3
eth-hash==0.2.0
eth-keyfile==0.5.1
eth-keys==0.2.4
eth-rlp==0.1.2
eth-typing==2.1.0
eth-utils==1.7.0
fastecdsa==1.7.4
hexbytes==0.2.0
idna==2.8
ifaddr==0.1.6
ipython==7.8.0
ipython-genutils==0.2.0
jedi==0.15.1
jsonschema==3.0.1
lahja==0.14.5
libp2p==0.1.1
lru-dict==1.1.6
lxml==4.4.1
milagro-bls-binding==0.1.3
multiaddr==0.0.8
multidict==4.5.2
mypy==0.740
mypy-extensions==0.4.3
netaddr==0.7.19
netdisco==2.6.0
netifaces==0.10.9
outcome==1.0.1
parsimonious==0.8.1
parso==0.5.1
pexpect==4.7.0
pickleshare==0.7.5
plyvel==1.1.0
prompt-toolkit==2.0.10
protobuf==3.9.0
ptyprocess==0.6.0
py-ecc==1.7.1
py-evm==0.3.0a7
pycparser==2.19
pycryptodome==3.9.0
pyethash==0.1.27
Pygments==2.4.2
pymultihash==0.8.2
pyrsistent==0.15.4
pysha3==1.0.2
python-dateutil==2.8.0
python-snappy==0.5.4
requests==2.22.0
rlp==1.1.0
rpcudp==3.0.0
ruamel.yaml==0.15.98
six==1.12.0
sniffio==1.1.0
sortedcontainers==2.1.0
SQLAlchemy==1.3.10
ssz==0.1.5
termcolor==1.1.0
toolz==0.10.0
traitlets==4.3.3
trie==1.4.0
trio==0.11.0
trio-typing==0.2.0
typed-ast==1.4.0
typing-extensions==3.7.4
u-msgpack-python==2.5.2
uPnPClient==0.0.8
urllib3==1.25.6
uvloop==0.11.2
varint==1.0.2
virtualenv==16.7.6
wcwidth==0.1.7
web3==4.4.1
websockets==5.0.1
yarl==1.3.0
zeroconf==0.23.0
```
### What is wrong?
Here is the log after running `pip3 install -U trinity`
```
...
Requirement already satisfied: ifaddr in /usr/local/lib/python3.7/site-packages (from zeroconf>=0.21.0->netdisco->upnpclient<1,>=0.0.8->trinity) (0.1.6)
Building wheels for collected packages: plyvel
  Building wheel for plyvel (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /usr/local/opt/python/bin/python3.7 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/m_/h73qx1jn1tzg89dvc4m8457w0000gp/T/pip-install-vbtul44m/plyvel/setup.py'"'"'; __file__='"'"'/private/var/folders/m_/h73qx1jn1tzg89dvc4m8457w0000gp/T/pip-install-vbtul44m/plyvel/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /private/var/folders/m_/h73qx1jn1tzg89dvc4m8457w0000gp/T/pip-wheel-ipjjye75 --python-tag cp37
       cwd: /private/var/folders/m_/h73qx1jn1tzg89dvc4m8457w0000gp/T/pip-install-vbtul44m/plyvel/
  Complete output (39 lines):
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib.macosx-10.13-x86_64-3.7
  creating build/lib.macosx-10.13-x86_64-3.7/plyvel
  copying plyvel/_version.py -> build/lib.macosx-10.13-x86_64-3.7/plyvel
  copying plyvel/__init__.py -> build/lib.macosx-10.13-x86_64-3.7/plyvel
  running build_ext
  building 'plyvel._plyvel' extension
  creating build/temp.macosx-10.13-x86_64-3.7
  creating build/temp.macosx-10.13-x86_64-3.7/plyvel
  clang -Wno-unused-result -Wsign-compare -Wunreachable-code -fno-common -dynamic -DNDEBUG -g -fwrapv -O3 -Wall -I/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk/usr/include -I/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk/System/Library/Frameworks/Tk.framework/Versions/8.5/Headers -I/usr/local/include -I/usr/local/opt/openssl@1.1/include -I/usr/local/opt/sqlite/include -I/usr/local/Cellar/python/3.7.4_1/Frameworks/Python.framework/Versions/3.7/include/python3.7m -c plyvel/_plyvel.cpp -o build/temp.macosx-10.13-x86_64-3.7/plyvel/_plyvel.o -Wall -g -mmacosx-version-min=10.7 -stdlib=libc++
  In file included from plyvel/_plyvel.cpp:589:
  In file included from /usr/local/include/leveldb/db.h:12:
  In file included from /usr/local/include/leveldb/iterator.h:20:
  /usr/local/include/leveldb/status.h:27:11: error: expected ';' at end of declaration list
    Status() noexcept : state_(nullptr) {}
            ^
  /usr/local/include/leveldb/status.h:33:23: error: expected ';' at end of declaration list
    Status(Status&& rhs) noexcept : state_(rhs.state_) { rhs.state_ = nullptr; }
                        ^
  /usr/local/include/leveldb/status.h:103:16: error: 'Status' is missing exception specification 'throw()'
  inline Status::Status(const Status& rhs) {
                 ^
  /usr/local/include/leveldb/status.h:24:22: note: previous declaration is here
  class LEVELDB_EXPORT Status {
                       ^
  /usr/local/include/leveldb/status.h:115:48: error: expected function body after function declarator
  inline Status& Status::operator=(Status&& rhs) noexcept {
                                                 ^
  plyvel/_plyvel.cpp:21159:125: warning: offset of on non-POD type 'struct __pyx_obj_6plyvel_7_plyvel_Iterator' [-Winvalid-offsetof]
    if (__pyx_type_6plyvel_7_plyvel_Iterator.tp_weaklistoffset == 0) __pyx_type_6plyvel_7_plyvel_Iterator.tp_weaklistoffset = offsetof(struct __pyx_obj_6plyvel_7_plyvel_Iterator, __pyx_base.__weakref__);
                                                                                                                              ^                                                    ~~~~~~~~~~
  /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/9.1.0/include/stddef.h:120:24: note: expanded from macro 'offsetof'
  #define offsetof(t, d) __builtin_offsetof(t, d)
                         ^                     ~
  1 warning and 4 errors generated.
  error: command 'clang' failed with exit status 1
  ----------------------------------------
  ERROR: Failed building wheel for plyvel
  Running setup.py clean for plyvel
Failed to build plyvel
Installing collected packages: cached-property, asyncio-cancel-token, netifaces, plyvel, pyethash, trie, eth-bloom, py-evm, trinity
  Found existing installation: plyvel 1.1.0
    Uninstalling plyvel-1.1.0:
      Successfully uninstalled plyvel-1.1.0
    Running setup.py install for plyvel ... error
    ERROR: Command errored out with exit status 1:
     command: /usr/local/opt/python/bin/python3.7 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/m_/h73qx1jn1tzg89dvc4m8457w0000gp/T/pip-install-vbtul44m/plyvel/setup.py'"'"'; __file__='"'"'/private/var/folders/m_/h73qx1jn1tzg89dvc4m8457w0000gp/T/pip-install-vbtul44m/plyvel/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /private/var/folders/m_/h73qx1jn1tzg89dvc4m8457w0000gp/T/pip-record-lbmircnb/install-record.txt --single-version-externally-managed --compile
         cwd: /private/var/folders/m_/h73qx1jn1tzg89dvc4m8457w0000gp/T/pip-install-vbtul44m/plyvel/
    Complete output (39 lines):
    running install
    running build
    running build_py
    creating build
    creating build/lib.macosx-10.13-x86_64-3.7
    creating build/lib.macosx-10.13-x86_64-3.7/plyvel
    copying plyvel/_version.py -> build/lib.macosx-10.13-x86_64-3.7/plyvel
    copying plyvel/__init__.py -> build/lib.macosx-10.13-x86_64-3.7/plyvel
    running build_ext
    building 'plyvel._plyvel' extension
    creating build/temp.macosx-10.13-x86_64-3.7
    creating build/temp.macosx-10.13-x86_64-3.7/plyvel
    clang -Wno-unused-result -Wsign-compare -Wunreachable-code -fno-common -dynamic -DNDEBUG -g -fwrapv -O3 -Wall -I/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk/usr/include -I/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk/System/Library/Frameworks/Tk.framework/Versions/8.5/Headers -I/usr/local/include -I/usr/local/opt/openssl@1.1/include -I/usr/local/opt/sqlite/include -I/usr/local/Cellar/python/3.7.4_1/Frameworks/Python.framework/Versions/3.7/include/python3.7m -c plyvel/_plyvel.cpp -o build/temp.macosx-10.13-x86_64-3.7/plyvel/_plyvel.o -Wall -g -mmacosx-version-min=10.7 -stdlib=libc++
    In file included from plyvel/_plyvel.cpp:589:
    In file included from /usr/local/include/leveldb/db.h:12:
    In file included from /usr/local/include/leveldb/iterator.h:20:
    /usr/local/include/leveldb/status.h:27:11: error: expected ';' at end of declaration list
      Status() noexcept : state_(nullptr) {}
              ^
    /usr/local/include/leveldb/status.h:33:23: error: expected ';' at end of declaration list
      Status(Status&& rhs) noexcept : state_(rhs.state_) { rhs.state_ = nullptr; }
                          ^
    /usr/local/include/leveldb/status.h:103:16: error: 'Status' is missing exception specification 'throw()'
    inline Status::Status(const Status& rhs) {
                   ^
    /usr/local/include/leveldb/status.h:24:22: note: previous declaration is here
    class LEVELDB_EXPORT Status {
                         ^
    /usr/local/include/leveldb/status.h:115:48: error: expected function body after function declarator
    inline Status& Status::operator=(Status&& rhs) noexcept {
                                                   ^
    plyvel/_plyvel.cpp:21159:125: warning: offset of on non-POD type 'struct __pyx_obj_6plyvel_7_plyvel_Iterator' [-Winvalid-offsetof]
      if (__pyx_type_6plyvel_7_plyvel_Iterator.tp_weaklistoffset == 0) __pyx_type_6plyvel_7_plyvel_Iterator.tp_weaklistoffset = offsetof(struct __pyx_obj_6plyvel_7_plyvel_Iterator, __pyx_base.__weakref__);
                                                                                                                                ^                                                    ~~~~~~~~~~
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/9.1.0/include/stddef.h:120:24: note: expanded from macro 'offsetof'
    #define offsetof(t, d) __builtin_offsetof(t, d)
                           ^                     ~
    1 warning and 4 errors generated.
    error: command 'clang' failed with exit status 1
    ----------------------------------------
  Rolling back uninstall of plyvel
  Moving to /usr/local/lib/python3.7/site-packages/plyvel-1.1.0.dist-info/
   from /usr/local/lib/python3.7/site-packages/~lyvel-1.1.0.dist-info
  Moving to /usr/local/lib/python3.7/site-packages/plyvel/
   from /usr/local/lib/python3.7/site-packages/~lyvel
ERROR: Command errored out with exit status 1: /usr/local/opt/python/bin/python3.7 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/m_/h73qx1jn1tzg89dvc4m8457w0000gp/T/pip-install-vbtul44m/plyvel/setup.py'"'"'; __file__='"'"'/private/var/folders/m_/h73qx1jn1tzg89dvc4m8457w0000gp/T/pip-install-vbtul44m/plyvel/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /private/var/folders/m_/h73qx1jn1tzg89dvc4m8457w0000gp/T/pip-record-lbmircnb/install-record.txt --single-version-externally-managed --compile Check the logs for full command output.
```
it's related to [py-evm #1815](https://github.com/ethereum/py-evm/issues/1815)

### How can it be fixed
Changing to plyvel==1.1.0 in setup.py to fix the issue. (verify by running `pip3 install -e .`)

--- a/setup.py
+++ b/setup.py
@@ -34,7 +34,7 @@ deps = {
         "dataclasses>=0.6, <1;python_version<'3.7'",
         "eth-utils>=1.7,<2",
         "ipython>=7.8.0,<8.0.0",
-        "plyvel==1.0.5",
+        "plyvel==1.1.0",
         PYEVM_DEPENDENCY,
         "web3==4.4.1",
         "lahja>=0.14.5,<0.15.0",

